src/sql/parts/grid/views/editData/editDataGridActions.ts

```diff
@@ -56,18 +56,18 @@ export class DeleteRowAction extends Action {
 
 export class RevertRowAction extends Action {
 	public static ID = 'grid.revertRow';
-	public static LABEL = localize('revertRow', 'Revert Row');
+	public static LABEL = localize('revertRow', 'Revert Current Row');
```

Comment:
Does this mean there is no way to revert a row that isn't currently selected?

Reply:
Correct. There's no way to have edits in a row that is not currently selected as far as I'm aware, so let me know if you think that's possible but otherwise it shouldn't be a problem.

Reply:
I guess I always assumed you could revert changes you have made previously in the session, but maybe not.